plugins/woocommerce/src/Internal/Admin/Settings/PaymentsController.php

```diff
@@ -0,0 +1,209 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\Internal\Admin\Settings;
+
+use Automattic\WooCommerce\Admin\Features\Features;
+use Exception;
+
+defined( 'ABSPATH' ) || exit;
+/**
+ * Payments settings controller class.
+ *
+ * Use this class for hooks and actions related to the Payments settings page.
+ */
+class PaymentsController {
```

Comment:
I don't see much benefit from introducing unit tests for the methods here (at least not at this point). We should be pretty covered with manual testing.

Reply:
sounds good to me. It was tested except needs setup use case.